tests/network-tests/src/scenarios/carthageNPoSSwitch.ts

```diff
@@ -0,0 +1,10 @@
+// import changingAuthorities from 'src/flows/carthage/changingAuthorities'
+// import claimingPayoutsEnabled from 'src/flows/carthage/claimingPayoutEnabled'
+// import setForceNewEra from 'src/flows/carthage/setForceNewEra'
+// import currentEraIsSomeInPoS from 'src/flows/carthage/currentEraIsSomeInPoS'
+import { scenario } from '../Scenario'
+import switchToNPoS from '../flows/carthage/switchToNPoS'
+
+scenario('Carthage NPoS switch', async ({ job }) => {
+  job('switch to npos checks', switchToNPoS)
```

Comment:
shouldn't this scenario setup the validators and nominators with running the carthagePoAAssertions before doing the switch? Or will there be an issue with block production after switch if we don't run a second validator?